hack/lib/common.bash

```diff
@@ -15,16 +15,21 @@ function array.join {
 
 # Loops until duration (car) is exceeded or command (cdr) returns non-zero
 function timeout {
-  local seconds timeout interval
-  interval=5
+  local timeout
+  timeout="${1:?Pass a timeout as arg[1]}"
+  interval="${interval:-1}"
```

Comment:
Does this change the behavior in that there is no interval at all anymore? How do I set it? How does this interop with the `seconds=$(( seconds + interval ))` thing?

Reply:
It sets a variable to 1 if it isn't set.

You can override it by calling:

```bash
interval=5 timeout 300 "[[ \$(something --to-check) != 'True' ]]"
```

That will wait for 5 seconds instead of 1.